staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -723,11 +723,11 @@ func (b *throttledLogger) attemptToLog() (klog.Level, bool) {
 	return -1, false
 }
 
-// Infof will write a log message at each logLevel specified by the receiver's throttleSettings
+// Info will write a log message at each logLevel specified by the receiver's throttleSettings
 // as long as it hasn't written a log message more recently than minLogInterval.
-func (b *throttledLogger) Infof(message string, args ...interface{}) {
+func (b *throttledLogger) Info(ctx context.Context, message string) {
```

Comment:
this seems to use a different approach https://github.com/kubernetes/kubernetes/pull/127709/files

Reply:
Oh I didn't realize that there is an existing PR. Will close this one then.
/close